fhir-model/src/main/java/com/ibm/fhir/model/type/Base64Binary.java

```diff
@@ -33,7 +33,7 @@ private Base64Binary(Builder builder) {
 
     /**
      * The actual value
-     * 
+     *
```

Comment:
I'm happy to approve this... however... This is a generated class, perhaps the generator updates are missing?

Reply:
good call